lib/galaxy/tools/deps/resolvers/__init__.py

```diff
@@ -91,8 +92,12 @@ class NullDependency( Dependency ):
     dependency_type = None
     exact = True
 
+    def __init__(self, version, name):
+        self.version = version
+        self.name = name
+
     def shell_commands( self, requirement ):
         return None
 
 
-INDETERMINATE_DEPENDENCY = NullDependency()
+INDETERMINATE_DEPENDENCY = NullDependency
```

Comment:
Its fine to switch it like this - but I'd take @nsoranzo's advice and use NullDependency instead of INDETERMINATE_DEPENDENCY - the latter isn't a valid class name according to most Python naming conventions.